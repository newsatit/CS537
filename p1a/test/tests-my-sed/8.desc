another long lines of text
