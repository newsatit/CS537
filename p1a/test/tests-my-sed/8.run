./my-sed text "" INPUT_DIR/8.in
