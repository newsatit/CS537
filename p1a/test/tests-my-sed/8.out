these are very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  long lines of 
these are very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  long lines of 
these are very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  long lines of 
these are not
these are still not
these are really not
these are very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  very  very  very  very  very   very  very  very  very  very  long lines of .

